e2e/packages/wp-source-errors/src/index.tsx

```diff
@@ -23,7 +23,8 @@ const wpSourceErrors: WpSourceErrors = {
       init: ({ state, libraries }) => {
         const { query } = libraries.source.parse(state.frontity.initialLink);
         if (query.statusCode) {
-          state.source.api = `http://httpstat.us/${query.statusCode}?rest_route=/`;
+          // This is a simple app made on glitch.me which just returns the HTTP status code passed as param.
+          state.source.api = `https://ballistic-western-donkey.glitch.me/status/${query.statusCode}?rest_route=/`;
```

Comment:
The tests were also failing now because httpstat.us is down 🙁 

https://github.com/Readify/httpstatus/issues/89

So, I've just made a quick app on glitch which does the same thing as https://httpstat.us and it works fine.

Reply:
Hey, I'm going to cherry-pick and add this to `dev` (tests are failing also in my PR :sweat_smile:).